nmrfx-processor-gui/src/main/java/org/nmrfx/processor/gui/log/ChangeLogLevelController.java

```diff
@@ -0,0 +1,139 @@
+package org.nmrfx.processor.gui.log;
+
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TextInputDialog;
+import javafx.scene.control.cell.ChoiceBoxTableCell;
+import javafx.stage.Modality;
+import javafx.stage.Stage;
+import javafx.stage.StageStyle;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.ResourceBundle;
+
+
+public class ChangeLogLevelController implements Initializable {
+
+    private static final String TITLE = "Change log level...";
+    private Stage stage;
+    @FXML
+    private TableView<Map.Entry<String, LogLevel>> changeLogLevelTable;
+
+    /**
+     * Creates a new ChangeLogLevelController
+     * @param parentStage The parent stage of the new ChangeLogLevelController
+     * @return ChangeLogLevelController
+     */
+    public static ChangeLogLevelController create(Stage parentStage) {
+        FXMLLoader loader = new FXMLLoader(ChangeLogLevelController.class.getResource("/fxml/ChangeLogLevelScene.fxml"));
+        Stage stage = new Stage(StageStyle.DECORATED);
+        ChangeLogLevelController controller;
+        try {
+            Scene scene = new Scene(loader.load());
+            stage.setScene(scene);
+            controller = loader.getController();
+            controller.stage = stage;
+            stage.setTitle(TITLE);
+            stage.setResizable(false);
+            stage.initOwner(parentStage);
+            stage.initModality(Modality.WINDOW_MODAL);
```

Comment:
When the ChangeLogLevelScene is showing, cannot interact with the log console until the popup is closed, so multiple ChangeLogLevelScenes cannot be opened, but will still be able to interact with the NMRFx main app.